spec/actions/create_record_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+RSpec.describe 'actions/create_record', :vcr do
+
+  let(:connector) { Workato::Connector::Sdk::Connector.from_file('connector.rb', settings) }
+  let(:settings) { Workato::Connector::Sdk::Settings.from_encrypted_file('settings.yaml.enc', 'master.key') }
+
+  let(:action) { connector.actions.create_record }
+
+  subject(:input) {
+    input = JSON.parse(File.read('fixtures/methods/post/input/create.json'))
+  }
+  subject(:output) { action.execute(settings, input) }
+
+  context 'execute' do
+
+    # Request Response
+    it 'is an object with success' do
+
+      ## General Response
+      expect(output).to be_kind_of(::Object)
```

Comment:
```suggestion
      expect(output).to be_kind_of(::Hash)
```

 `::Object` is a base class of almost everything, "Hash" is better for hash-like instances.